apple/common/apple_gfx_context.c.inl

```diff
@@ -241,8 +241,8 @@ static void apple_gfx_ctx_get_video_size(void *data, unsigned* width, unsigned*
 #endif
    }
 
-   *width  = CGRectGetWidth(size)  * screen.scale;
-   *height = CGRectGetHeight(size) * screen.scale;
+   *width  = CGRectGetWidth(size)  * screen.nativeScale;
+   *height = CGRectGetHeight(size) * screen.nativeScale;
```

Comment:
This is going to crash on anything lower than iOS 8.

You're going to want to check with something like this:

```objc
if ([screen respondsToSelector:@selector(nativeScale)]) {
  *width  = CGRectGetWidth(size)  * screen.nativeScale;
  *height = CGRectGetHeight(size) * screen.nativeScale;
}else{
  *width  = CGRectGetWidth(size)  * screen.scale;
  *height = CGRectGetHeight(size) * screen.scale;
}

```